package.json

```diff
@@ -30,18 +30,18 @@
     "@graphql-typed-document-node/core": "^3.1.0",
     "@types/aws4": "^1.5.1",
     "@types/jest": "^26.0.15",
-    "@types/node": "^14.14.10",
+    "@types/node": "^16.18.96",
```

Comment:
updating `@types/node` was necessary to get the types for `signal` support. This should _not_ change the minimum required node version of this project, as older versions of node will simply ignore the `signal` property.